card/mod/notifications/set/self/follow_defaults.rb

```diff
@@ -31,7 +31,8 @@
 
 format :html do
   view :edit, perms: :update, tags: :unknown_ok do
-    frame_and_form :update do
+    frame_and_form :update, hidden: { success: "_self",
+                                      card: { update_all_users: false } } do
```

Comment:
lord, that's much more readable.

Though I don't actually remember what `update_all_users` means...